tests/mocks/repository/lib/mock_kvset.c

```diff
@@ -367,7 +367,25 @@ _kvset_get_nth_vblock_id(struct kvset *kvset, uint32_t index)
 }
 
 static uint32_t
-_kvset_get_nth_vblock_len(struct kvset *kvset, uint32_t index)
+_kvset_get_nth_vblock_alen(struct kvset *kvset, uint32_t index)
+{
+    struct mock_kvset *mk = (void *)kvset;
+    struct kvdata *    iterv = mk->iter_data;
+    int                i, kvcnt, vcnt = 0;
+
+    if (index >= mk->stats.kst_vblks)
+        return 0;
+
+    kvcnt = iterv[0].key; /* number of elements */
+    for (i = 1; i <= kvcnt; i++)
+        if (iterv[i].val != -1) /* do not count tombstones */
+            vcnt++;
+
+    return vcnt * sizeof(int);
```

Comment:
I know this is a test.  But still.... it makes no sense to "alen" to care about tombstones.

Reply:
Obviously this is just a copy of the function beneath it. Why does wlen not count tombstones? Why does alen differ in this regard?

Reply:
This is a unit test.  The test probably  knows that values are integers and `wlen` is more or less `vcnt * sizeof(int)`.  In real code it would be the sum of lengths given to `mblock_write` when creating the mblock.

In real code, the `alen` would simply be the mblocks allocated length which can be significantly more than the number of bytes written to the mblock.

I looked at the test/mock code and can't see any reasonable value for `alen` (the mock doesn't seem to care).  But... it would be entirely reasonable for the mock to use `wlen` rounded up to `4K` or `32K` as the `alen`.